deploying-daml/daml-on-fabric/daml_run_json_api.md

```diff
@@ -0,0 +1,6 @@
+Okay now that our code is deployed if we wanted to we could talk directly to it using gRPC, but that's a bit cumbersome and not really what we want to build our user-facing applications against. Lucky for us our `daml` assistant also gives us the ability to automatically generate JSON-API endpoints based on our code. So we can just start up this service and our UI will have an API to consume.
+
```

Comment:
The process is done when you see the `17:23:29.292 [http-json-ledger-api-akka.actor.default-dispatcher-6] INFO  com.daml.http.Main$ - Started server: ServerBinding(/0:0:0:0:0:0:0:0:7575)` message in the terminal.